cardano-cli/src/Cardano/CLI/EraBased/Common/Option.hs

```diff
@@ -1803,26 +1805,44 @@ make' format desc flag_ extraHelp kind =
       , Opt.long ("output-" <> flag_)
       ]
 
-pFormatCBOR
-  :: FormatCBOR :| fs
-  => String
+parserFromFormatFlags
+  :: String
+  -> [Flag (Vary fs)]
   -> Parser (Vary fs)
-pFormatCBOR =
-  make' FormatCBOR "BASE16 CBOR" "cbor" Nothing
+parserFromFormatFlags content =
+  parserFromFlags $ \f ->
+    mconcat
+      [ "Format "
+      , content
+      , " to "
+      , f.format
+      , if f.options.isDefault then " (default)" else ""
```

Comment:
I'm using `OverloadedRecordDot` to allow me to use record dot syntax like this instead of `isDefault (options f)`.

However this isn't the main reason I like this extension.